frontend/src/app/modules/boards/board/inline-add/board-inline-add-autocompleter.html

```diff
@@ -1,17 +1,15 @@
-<ng-select ngClass="wp-inline-create--reference-autocompleter"
-           [items]="results$ | async"
-           [appendTo]="appendToContainer"
-           [multiple]="false"
-           [loading]="isLoading"
-           [typeahead]="searchInput$"
-           [closeOnSelect]="false"
-           (open)="opened()"
-           (close)="cancel()"
-           (change)="addWorkPackageToQuery($event)">
-  <ng-template ng-label-tmp let-item="item">
-    {{item.type.name }} #{{ item.id }} {{ item.subject }}
-  </ng-template>
-  <ng-template ng-option-tmp let-item="item" let-index="index" let-search="searchTerm">
-    {{item.type.name }} #{{ item.id }} {{ item.subject }}
-  </ng-template>
-</ng-select>
+<op-autocompleter
+[appendTo]="appendToContainer"
+[multiple]="false"
+[ngClass]="'wp-inline-create--reference-autocompleter'"
+[closeOnSelect]="false"
+[defaulData]="false"
+[placeholder]="text.placeholder"
+[loading]="isLoading"
+[resource]="autocompleterOptions.resource"
+[searchKey]="autocompleterOptions.searchKey"
+[getOptionsFn]="autocompleterOptions.getOptionsFn"
+(open)="opened()"
```

Comment:
Here we have a method with a setTimeout only to repositioning the ngSelect. This should probably be handled inside the component on every open if it is a common need or provide a @Input repositionOnOpen.